tests/boot/boot_from_pxe.pm

```diff
@@ -38,8 +38,7 @@ sub run() {
         #Numburg
         #send_key_until_needlematch "qa-net-selection-" . get_var('DISTRI') . "-" . get_var("VERSION"), 'down', 30, 3;
         #Don't use send_key_until_needlematch to pick first menu tier as dist network sources might not be ready when openQA is running tests
-        send_key 'esc';
-        assert_screen 'qa-net-boot';
+        send_key_until_needlematch 'qa-net-boot', 'esc';
```

Comment:
while you are working on this please get in contact with the previous author of these lines and also update the previous lines, e.g. s/Numburg/Nuremberg/, delete disabled code and update the comment.

I'm not sure if it's feasible to provide verification runs. Might be worth to try. You could maybe talk to admins to make the worker a shared one. In any case after merge please make sure to monitor virtualization jobs that rely on this AFAIU.